lib/integrations/awsoidc/deploydatabaseservice_test.go

```diff
@@ -20,12 +20,18 @@ package awsoidc
 
 import (
 	"context"
+	"fmt"
+	"net/http"
+	"sync"
 	"testing"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
+	awshttp "github.com/aws/aws-sdk-go-v2/aws/transport/http"
 	"github.com/aws/aws-sdk-go-v2/service/ecs"
 	ecsTypes "github.com/aws/aws-sdk-go-v2/service/ecs/types"
 	"github.com/aws/aws-sdk-go-v2/service/sts"
+	awsV1Http "github.com/aws/smithy-go/transport/http"
```

Comment:
Nit: this really should be awsv1http. It's not common to miss case in package names or aliases.